### PR TITLE
Keywords2: Update whitelisted Keywords

### PR DESCRIPTION
**Pull request by manticore-projects**
Updates the Keywords in `RelObjectNameWithoutValue` after PR #1649 introduced new tokens.

Workflow should be like that:

1) Amend Tokens in the Grammar
2) When keyword-tests fail, run `gradle updateKeywords`
3) Then build via `mvn package` or `gradle build` (it does not matter)
4) Push/Deploy

Using `gradle updateKeywords` task is optional, you can also add the new Tokens manually has it has always been done before.

After accepting the Keywords PR, any further PR will automatically fail acceptance tests, unless keywords are in order.

### CODE REVIEW

**Comment by wumpz** [comment]
Since I have accepted those enhanced keywords, why is this pr so large?

**Comment by manticore-projects** [comment]
> Since I have accepted those enhanced keywords, why is this pr so large?

It actually changes only 4 Files:
1) the Grammar, fixing the `RelObjectNameWithoutValue()` keywords
2) the Maven Build file, adding a Task for updating the `RelObjectNameWithoutValue()` keywords and dependencies
3) the Gradle Build file, updating dependencies
4) Oracle Special Tests, since Keywords enabled 2 more tests

I do not know, why GIT carries all the change set since branching, after I merged MASTER.

File: pom.xml [hunk 1]
```diff
@@ -114,6 +116,18 @@
 
     <build>
         <plugins>
+            <!-- Update they Keyword Whitelist in the Grammar's production RelObjectNameWithoutValue() -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```
Comment (wumpz) [inline pom.xml:121]: So how does this actually is processed? I am not sure how this exec-maven-plugin is activated in the maven build process?

Comment (wumpz) [reply pom.xml:121]: got it from the other PR

**Pull request merged**